### ISSUE TITLE
Rpgitems Powerloss

### ISSUE THREAD

**Issue by Midnight9746** [NyaaCat/RPGitems-reloaded#306]
I updated to version 3.5.314, (Spigot 1.12.2) and the one before and rpgitems keep losing their powers now. No matter what. I added them all back and just when I finished the first ones I fixed relost their powers. It's wierd.

**Comment by Midnight9746** [comment]
@Librazy, it's also setting armour % to higher values. Like say 31% goes to 160%

**Comment by Librazy** [comment]
Any NullPointerException in console?

**Comment by Midnight9746** [comment]
All I'm seeing is "Unknown translation" for the language file

**Comment by Librazy** [comment]
Unknown translation should not be a major problem. Setting up local test environment trying to reproduce you issue now.

**Comment by Midnight9746** [comment]
@Librazy Essentially, I updated the plugin, restarted the server. Then all the items were fine, but when I came back the next day, they had no powers. So I added them back, and when I got done, the first ones done a few hours ago resetted.

**Comment by Midnight9746** [comment]
Config (I keep deleting this file and it regens different versions so idk):

autoupdate: true
defaults:
  hand: One handed
  sword: Sword
  damage: Damage
  armour: Armour
support:
  worldguard: true
version: '0.5'
language: en_GB

**Comment by Librazy** [comment]
Are those power still in the items.yml ?

**Comment by Midnight9746** [comment]
No, it at first set all of them back to Power: {}, and now it's randomly doing it to a few

**Comment by Midnight9746** [comment]
@Librazy I finished fixing my items, and I'm stopping my server. Then I'ma add the fixed file into the folder by itself. I'm hoping that it would regenerate the config and language files to be updated. So that it would possibly stop doing the removal of power and messing with armor %'s,

**Comment by Librazy** [comment]
Did you used `/rpgitem clone` before? I made a mistake in previous implementation of it, making operation on cloned item affects origin item. It should be fixed in 3.5.315.

**Comment by Midnight9746** [comment]
Yes, I actually did.. I ended up doing /rpgitem item clone item and then it just says "NewItem already exist!" and it wouldn't let me remove said item. I figured this did something to it but wasn't sure. But I edited my file to remove the clone items.

**Comment by Librazy** [comment]
Sorry for the inconvenience, there is another bug that it always respond /item clone with already exist. It would be fixed in upcoming version.

**Comment by Midnight9746** [comment]
@Librazy Spigot doesn't have 3.5.315, only 3.5.15

and it's okay lol... at least I'm the kind of person how would go over a file of 100+ or so items and fix it.. heh

**Comment by Midnight9746** [comment]
Found it... https://github.com/NyaaCat/RPGitems-reloaded/releases heh

**Comment by Librazy** [comment]
I do apologize for the mess in previous releases and thank you for reporting it. Will update spigot releases asap.

**Comment by Midnight9746** [comment]
It's totally okay! Heh, I figured there was a bug in clone but thought "this might not even mean anything and probably get fixed when they noticed the command doesn't work" and then it happend. I'm just happy I'm not my friend who has over probably 400 items. I did warn them to backup, just to make sure.

**Comment by Librazy** [comment]
Any abnormal in the console? The clone bug shouldn't affect a restarted server.

**Comment by Midnight9746** [comment]
Watching it now after I uploaded the file, did /rpgitem reload, and restarted.

**Comment by Midnight9746** [comment]
@Librazy after reloading the files to take on the fixed file and restarting, it didn't reset this time when it started. I'll keep the fixed file just in case though.. heh
